CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ one.
 ## Pull requests ##
 
 If you choose to [submit a pull
-request](https://github.com/cisagov/log4j-affected-db/pulls), you will
+request](https://github.com/cisagov/log4j-affected-db/PULL-EXAMPLE.md), you will
```

Comment:
Please add a subsection to this section directing people to go to the `PULL-EXAMPLE.md` file for directions.

Something loosely like:

```markdown
### Creating a pull request ###

Instructions for creating a pull request using the GitHub Web UI can be found
in [`PULL-EXAMPLE.md`](PULL-EXAMPLE.md).
```

```suggestion
request](https://github.com/cisagov/log4j-affected-db/pulls), you will
```